Signal/src/view controllers/MessagesViewController.m

```diff
@@ -918,6 +925,47 @@ - (OWSDisplayedMessageCollectionViewCell *)loadErrorMessageCellForMessage:(TSMes
 
 #pragma mark - Adjusting cell label heights
 
+
+/**
+ Due to the usage of JSQMessagesViewController, and it non-conformity to Dynamyc Type
+ we're left to our own devices to make this as usable as possible.
+ JSQMessagesVC also does not expose the constraint for the input toolbar height nor does it seem to
+ give us a method to tell it to re-adjust (I think it should observe the preferredDefaultHeight property).
```

Comment:
Not for long! https://github.com/WhisperSystems/Signal-iOS/pull/1474

Note that we are using the [stable branch](https://github.com/jessesquires/JSQMessagesViewController/tree/release_7.3) on upstream as opposed to development.